error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -788,4 +789,36 @@ Stream<T> after(T e1, T e2, T e3, T e4, T e5) {
       return Stream.of(e1, e2, e3, e4, e5);
     }
   }
+
+  /**
+   * Prefer {@link Streams#stream(Iterable)} over manually wrapping an {@link Iterable}
+   * with {@link StreamSupport#stream(java.util.Spliterator, boolean)} using {@code parallel = false}.
+   */
+  static final class IterableStream<T> {
```

Comment:
By convention, we name the refaster rule class after the result of the rewriting process:
```suggestion
  static final class StreamsStream<T> {
```

Reply:
(Naming is hard; I am sure others can chime in here)